sdcm/cluster.py

```diff
@@ -2080,7 +2082,6 @@ def install_scylla(self, scylla_repo, scylla_version=None):
 
         self.download_scylla_repo(scylla_repo)
         # TODO: consider moving this to cloud-init, since it's has nothing todo with scylla
-        self.install_package(package_name="rsync")
```

Comment:
The TODO comment above is tied to this removed line.
So, if remove, then both at once.

Reply:
Right,I was clearing it within the search window and didn't notice it